app/src/main/java/com/sun/cookbook/data/source/remote/fetchjson/ParseDataWithJson.kt

```diff
@@ -50,6 +54,37 @@ class ParseDataWithJson {
                         item?.let { data.add(item) }
                     }
                 }
+                TypeModel.RECIPE_DETAIL -> {
+                    parseJsonToObject(jsonObject, typeModel)?.let {
+                        return it
```

Comment:
Đoạn này logic là như nào nhỉ? 
Nếu không null thì return parsedObject 
Nếu null (parse lỗi) thì return empty list ?

Reply:
Dạ đúng rồi a,
Đoạn này e hơi bí vì dữ liệu trả về của hàm này là 1 list, còn API Detail là trả về một object, nên em mới làm kiểu này

Reply:
thế thì chức năng 2 hàm này bị lẫn lộn rồi 
1 hàm để parse jsonObject to List 
1 hàm để parse to Object 
Mỗi hàm chỉ 1 chức năng thôi chứ ?

Reply:
Anh review lại thử em làm theo cách khác có được không ạ, vì nếu 2 hàm riêng thì lúc gọi bên hàm onPostExcute của AsynTask thì làm sao gọi được ạ !